pkg/util/jvm/keystore_test.go

```diff
@@ -29,15 +31,20 @@ func TestGenerateKeystore(t *testing.T) {
 	// No Data
 	var data [][]byte
 	ctx := context.Background()
-	err := GenerateKeystore(ctx, "", "name", NewKeystorePassword(), data)
+	err := GenerateKeystore(ctx, "", "/tmp/keystore", NewKeystorePassword(), data)
 	assert.Nil(t, err)
 
 	// Correct input
 	data = [][]byte{{0}, {1}}
-	err = GenerateKeystore(ctx, "", "name", NewKeystorePassword(), data)
+	err = GenerateKeystore(ctx, "", "/tmp/keystore", NewKeystorePassword(), data)
 	assert.NotNil(t, err)
```

Comment:
I know it is out if scope, but I am really confused about the fact that we expect an error on something called "Correct input".

Reply:
I guess the comment is unfortunate and it should have been something like non-nil data.

Reply:
I will take a look at that later this week, the comment is a leftover. Can create another PR with it rephrased

Reply:
Yes, no rush though.